### ISSUE TITLE
Install error

### ISSUE THREAD

**Issue by nickeubank** [conda-forge/geopandas-feedstock#23]
Error on import:

    import geopandas as gpd 
    import pandas as pd 
    
    
    
    Traceback (most recent call last):
    
      File "<ipython-input-2-0419fb6eaddf>", line 1, in <module>
        import geopandas as gpd
    
      File "/Users/Nick/anaconda/lib/python3.6/site-packages/geopandas/__init__.py", line 4, in <module>
        from geopandas.io.file import read_file
    
      File "/Users/Nick/anaconda/lib/python3.6/site-packages/geopandas/io/file.py", line 3, in <module>
        import fiona
    
      File "/Users/Nick/anaconda/lib/python3.6/site-packages/fiona/__init__.py", line 69, in <module>
        from fiona.collection import Collection, BytesCollection, vsi_path
    
      File "/Users/Nick/anaconda/lib/python3.6/site-packages/fiona/collection.py", line 9, in <module>
        from fiona.ogrext import Iterator, ItemsIterator, KeysIterator
    
    ImportError: dlopen(/Users/Nick/anaconda/lib/python3.6/site-packages/fiona/ogrext.cpython-36m-darwin.so, 2): Library not loaded: @rpath/libpng16.16.dylib
      Referenced from: /Users/Nick/anaconda/lib/libgdal.20.dylib
      Reason: Incompatible library version: libgdal.20.dylib requires version 45.0.0 or later, but libpng16.16.dylib provides version 44.0.0

Install string from terminal:

    ~ conda install geopandas -c conda-forge
    Fetching package metadata ...........
    Solving package specifications: .
    
    Package plan for installation in environment /Users/Nick/anaconda:
    
    The following NEW packages will be INSTALLED:
    
        click-plugins:   1.0.3-py36_0       conda-forge
        cligj:           0.4.0-py36_0       conda-forge
        descartes:       1.1.0-py36_0       conda-forge
        expat:           2.1.0-2            conda-forge
        fiona:           1.7.5-np111py36_0  conda-forge
        freexl:          1.0.2-1            conda-forge
        gdal:            2.1.3-np111py36_0  conda-forge
        geopandas:       0.2.1-py36_3       conda-forge
        geos:            3.5.1-1            conda-forge
        giflib:          5.1.4-0            conda-forge
        hdf4:            4.2.12-0           conda-forge
        json-c:          0.12-0             conda-forge
        kealib:          1.4.6-3            conda-forge
        libdap4:         3.18.3-2           conda-forge
        libnetcdf:       4.4.1.1-2          conda-forge
        libpq:           9.5.4-3            conda-forge
        libspatialindex: 1.8.5-1            conda-forge
        libspatialite:   4.3.0a-14          conda-forge
        munch:           2.1.1-py36_0       conda-forge
        openjpeg:        2.1.2-2            conda-forge
        pcre:            8.39-0             conda-forge
        proj4:           4.9.3-3            conda-forge
        psycopg2:        2.6.2-py36_1       conda-forge
        pyproj:          1.9.5.1-py36_0     conda-forge
        pysal:           1.13.0-py36_0      conda-forge
        rtree:           0.8.3-py36_0       conda-forge
        shapely:         1.5.17-np111py36_2 conda-forge
        xerces-c:        3.1.4-0            defaults   
    
    The following packages will be SUPERCEDED by a higher-priority channel:
    
        conda:           4.3.14-py36_0      defaults    --> 4.2.13-py36_0 conda-forge
        conda-env:       2.6.0-0            defaults    --> 2.6.0-0       conda-forge
    
    Proceed ([y]/n)? y
    
    conda-env-2.6. 100% |################################| Time: 0:00:00 801.10 kB/s
    expat-2.1.0-2. 100% |################################| Time: 0:00:00 526.81 kB/s
    geos-3.5.1-1.t 100% |################################| Time: 0:00:01   1.87 MB/s
    giflib-5.1.4-0 100% |################################| Time: 0:00:00  14.43 MB/s
    json-c-0.12-0. 100% |################################| Time: 0:00:00 177.00 kB/s
    libspatialinde 100% |################################| Time: 0:00:00   1.46 MB/s
    pcre-8.39-0.ta 100% |################################| Time: 0:00:00   1.93 MB/s
    proj4-4.9.3-3. 100% |################################| Time: 0:00:01   2.10 MB/s
    xerces-c-3.1.4 100% |################################| Time: 0:00:00   4.50 MB/s
    freexl-1.0.2-1 100% |################################| Time: 0:00:00 590.77 kB/s
    hdf4-4.2.12-0. 100% |################################| Time: 0:00:00   1.40 MB/s
    libpq-9.5.4-3. 100% |################################| Time: 0:00:00 292.90 kB/s
    kealib-1.4.6-3 100% |################################| Time: 0:00:00 911.89 kB/s
    libdap4-3.18.3 100% |################################| Time: 0:00:02   1.22 MB/s
    libnetcdf-4.4. 100% |################################| Time: 0:00:01   1.36 MB/s
    libspatialite- 100% |################################| Time: 0:00:04   1.26 MB/s
    openjpeg-2.1.2 100% |################################| Time: 0:00:00 900.22 kB/s
    psycopg2-2.6.2 100% |################################| Time: 0:00:00   1.17 MB/s
    rtree-0.8.3-py 100% |################################| Time: 0:00:00  10.66 MB/s
    click-plugins- 100% |################################| Time: 0:00:00   7.57 MB/s
    cligj-0.4.0-py 100% |################################| Time: 0:00:00   9.86 MB/s
    conda-4.2.13-p 100% |################################| Time: 0:00:00   1.20 MB/s
    gdal-2.1.3-np1 100% |################################| Time: 0:00:08   2.20 MB/s
    munch-2.1.1-py 100% |################################| Time: 0:00:00   7.79 MB/s
    pyproj-1.9.5.1 100% |################################| Time: 0:00:01   1.89 MB/s
    shapely-1.5.17 100% |################################| Time: 0:00:00   2.74 MB/s
    fiona-1.7.5-np 100% |################################| Time: 0:00:00   1.90 MB/s
    pysal-1.13.0-p 100% |################################| Time: 0:00:07   1.76 MB/s
    descartes-1.1. 100% |################################| Time: 0:00:00   2.43 MB/s
    geopandas-0.2. 100% |################################| Time: 0:00:00   1.04 MB/s

**Comment by jorisvandenbossche** [comment]
On Ubuntu 16.04, in a fresh environment (`conda create -n geopandas-test geopandas -c conda-forge`), importing geopandas works for me

**Comment by nickeubank** [comment]
Thanks @jorisvandenbossche . Interesting. Works in clean install. Which is odd, because I JUST clean-installed anaconda on this computer. Something in the default anaconda install environment conflicting? Anyone know command to create a new environment with anaconda default installs? Might be worth figuring out conflict. 

Round 2 with clean install:

    conda create -n geopandas-test-mac python=3.6 geopandas -c conda-forge
    Fetching package metadata .........
    Solving package specifications: ..........
    
    Package plan for installation in environment /Users/Nick/anaconda/envs/geopandas-test-mac:
    
    The following packages will be downloaded:
    
        package                    |            build
        ---------------------------|-----------------
        ca-certificates-2017.1.23  |                0         166 KB  conda-forge
        icu-58.1                   |                1        22.7 MB  conda-forge
        jpeg-9b                    |                0         243 KB  conda-forge
        libgfortran-3.0.0          |                0         465 KB  conda-forge
        libiconv-1.14              |                4         1.3 MB  conda-forge
        ncurses-5.9                |               10         1.1 MB  conda-forge
        sqlite-3.13.0              |                1         1.4 MB  conda-forge
        tk-8.5.19                  |                1         2.1 MB  conda-forge
        xz-5.2.2                   |                0         288 KB  conda-forge
        zlib-1.2.11                |                0          95 KB  conda-forge
        hdf5-1.8.17                |                9         4.0 MB  conda-forge
        libtiff-4.0.6              |                7         556 KB  conda-forge
        libxml2-2.9.4              |                4         1.9 MB  conda-forge
        openssl-1.0.2h             |                3         3.3 MB  conda-forge
        readline-6.2               |                0         311 KB  conda-forge
        xerces-c-3.1.4             |                3         3.4 MB  conda-forge
        curl-7.52.1                |                0         533 KB  conda-forge
        freetype-2.7               |                1         771 KB  conda-forge
        python-3.6.1               |                0        12.0 MB  conda-forge
        certifi-2017.1.23          |           py36_0         222 KB  conda-forge
        click-6.7                  |           py36_0         103 KB  conda-forge
        numpy-1.12.1               |           py36_0         3.5 MB  defaults
        pyparsing-2.2.0            |           py36_0          95 KB  conda-forge
        pytz-2016.10               |           py36_0         189 KB  conda-forge
        six-1.10.0                 |           py36_1          18 KB  conda-forge
        sqlalchemy-1.1.5           |           py36_0         1.5 MB  conda-forge
        tornado-4.4.2              |           py36_0         582 KB  conda-forge
        cycler-0.10.0              |           py36_0          14 KB  conda-forge
        gdal-2.1.3                 |      np112py36_2        17.8 MB  conda-forge
        python-dateutil-2.6.0      |           py36_0         232 KB  conda-forge
        scipy-0.19.0               |      np112py36_0        15.0 MB  defaults
        setuptools-33.1.1          |           py36_0         502 KB  conda-forge
        shapely-1.5.17             |      np112py36_2         212 KB  conda-forge
        fiona-1.7.5                |      np112py36_0         405 KB  conda-forge
        matplotlib-2.0.0           |      np112py36_3         6.7 MB  conda-forge
        pandas-0.19.2              |      np112py36_1         8.7 MB  conda-forge
        wheel-0.29.0               |           py36_0          82 KB  conda-forge
        pip-9.0.1                  |           py36_0         1.6 MB  conda-forge
        ------------------------------------------------------------
                                               Total:       113.9 MB
    
    The following NEW packages will be INSTALLED:
    
        ca-certificates: 2017.1.23-0        conda-forge
        certifi:         2017.1.23-py36_0   conda-forge
        click:           6.7-py36_0         conda-forge
        click-plugins:   1.0.3-py36_0       conda-forge
        cligj:           0.4.0-py36_0       conda-forge
        curl:            7.52.1-0           conda-forge
        cycler:          0.10.0-py36_0      conda-forge
        descartes:       1.1.0-py36_0       conda-forge
        expat:           2.1.0-2            conda-forge
        fiona:           1.7.5-np112py36_0  conda-forge
        freetype:        2.7-1              conda-forge
        freexl:          1.0.2-1            conda-forge
        gdal:            2.1.3-np112py36_2  conda-forge
        geopandas:       0.2.1-py36_3       conda-forge
        geos:            3.5.1-1            conda-forge
        giflib:          5.1.4-0            conda-forge
        hdf4:            4.2.12-0           conda-forge
        hdf5:            1.8.17-9           conda-forge
        icu:             58.1-1             conda-forge
        jpeg:            9b-0               conda-forge
        json-c:          0.12-0             conda-forge
        kealib:          1.4.6-3            conda-forge
        libdap4:         3.18.3-2           conda-forge
        libgfortran:     3.0.0-0            conda-forge
        libiconv:        1.14-4             conda-forge
        libnetcdf:       4.4.1.1-2          conda-forge
        libpng:          1.6.28-0           conda-forge
        libpq:           9.5.4-3            conda-forge
        libspatialindex: 1.8.5-1            conda-forge
        libspatialite:   4.3.0a-14          conda-forge
        libtiff:         4.0.6-7            conda-forge
        libxml2:         2.9.4-4            conda-forge
        matplotlib:      2.0.0-np112py36_3  conda-forge
        mkl:             2017.0.1-0         defaults   
        munch:           2.1.1-py36_0       conda-forge
        ncurses:         5.9-10             conda-forge
        numpy:           1.12.1-py36_0      defaults   
        openjpeg:        2.1.2-2            conda-forge
        openssl:         1.0.2h-3           conda-forge
        pandas:          0.19.2-np112py36_1 conda-forge
        pcre:            8.39-0             conda-forge
        pip:             9.0.1-py36_0       conda-forge
        proj4:           4.9.3-3            conda-forge
        psycopg2:        2.6.2-py36_1       conda-forge
        pyparsing:       2.2.0-py36_0       conda-forge
        pyproj:          1.9.5.1-py36_0     conda-forge
        pysal:           1.13.0-py36_0      conda-forge
        python:          3.6.1-0            conda-forge
        python-dateutil: 2.6.0-py36_0       conda-forge
        pytz:            2016.10-py36_0     conda-forge
        readline:        6.2-0              conda-forge
        rtree:           0.8.3-py36_0       conda-forge
        scipy:           0.19.0-np112py36_0 defaults   
        setuptools:      33.1.1-py36_0      conda-forge
        shapely:         1.5.17-np112py36_2 conda-forge
        six:             1.10.0-py36_1      conda-forge
        sqlalchemy:      1.1.5-py36_0       conda-forge
        sqlite:          3.13.0-1           conda-forge
        tk:              8.5.19-1           conda-forge
        tornado:         4.4.2-py36_0       conda-forge
        wheel:           0.29.0-py36_0      conda-forge
        xerces-c:        3.1.4-3            conda-forge
        xz:              5.2.2-0            conda-forge
        zlib:            1.2.11-0           conda-forge
    
    Proceed ([y]/n)? y

[...]
    
    ~ source activate geopandas-test-mac
    (geopandas-test-mac) ➜  ~ python
    Python 3.6.1 | packaged by conda-forge | (default, Mar 23 2017, 21:57:00) 
    [GCC 4.2.1 Compatible Apple LLVM 6.1.0 (clang-602.0.53)] on darwin
    Type "help", "copyright", "credits" or "license" for more information.
    >>> import geopandas as gpd
    >>> a = gpd.GeoDataFrame()
    >>>

**Comment by nickeubank** [comment]
@jorvisvandenbossche can you install in your root anaconda directory?

**Comment by jorisvandenbossche** [comment]
Sorry, I have miniconda, so also my root install is not a full anaconda

Op 5-apr.-2017 3:56 AM schreef "Nick Eubank" <notifications@github.com>:

**Comment by nickeubank** [comment]
Update: Got to work in root directory of a friends mac. I think it may have to do with a previous install of GDAL on my computer (for use in R), suggesting something is different in the path between when its in the root directory and when it's in an environment. 

I now have `/Users/Nick/anaconda/bin:/Users/Nick/anaconda` at the head of my path (added to .bash_profile), and shows up when I echo or check in Python with os.environ['PATH'].  Previously it had been `/opt/local/bin:/opt/local/sbin:/Users/Nick/anaconda/bin`. But even after running `conda uninstall gdal geopandas` then reinstalling, no fix. 

Suggestions welcome!

**Comment by nickeubank** [comment]
Think this is Fiona, so moving to https://github.com/conda-forge/fiona-feedstock/issues/40

**Comment by ocefpaf** [comment]
> Think this is Fiona, so moving to conda-forge/fiona-feedstock#40

This is not `fiona`'s either. It seems to be particular to your setup.

**Comment by benjaminsingleton** [comment]
I have this same issue. I just restored my Mac running MacOS Sierra 10.12.6 and installed Anaconda 4.4.0. The first two things I did was `conda config --add channels conda-forge` and then `conda install -c conda-forge geopandas`. I then started IPython and when I typed `import geopandas`, I get the following error message:

```
ImportError: dlopen(/Users/benjamin/anaconda/lib/python3.6/site-packages/fiona/ogrext.cpython-36m-darwin.so, 2): Library not loaded: @rpath/libpng16.16.dylib
  Referenced from: /Users/benjamin/anaconda/lib/libgdal.20.dylib
  Reason: Incompatible library version: libgdal.20.dylib requires version 45.0.0 or later, but libpng16.16.dylib provides version 44.0.0
```

Per some of the comments I've read, I created a new environment `conda create --name geo`, activated the environment `source activate geo` and then installed `conda install -c conda-forge geopandas` again to no success.

So, I uninstalled Anaconda completely and instead installed Miniconda. The first thing I did after installing Miniconda was run `conda config --add channels conda-forge` and then `conda install -c conda-forge geopandas`. This worked, however I still got the following error message in IPython immediately after `import geopandas as gpd`: 

```
Failed `CDLL(/Library/Frameworks/GEOS.framework/Versions/Current/GEOS)`
Failed `CDLL(/opt/local/lib/libgeos_c.dylib)`
```
I was able to read in a shapefile so it appears to be working, but this was honestly a painful experience getting it properly installed. Is there a way we can fix this so that geopandas can successfully install with the default distribution of Anaconda?

**Comment by jorisvandenbossche** [comment]
>  I created a new environment conda create --name geo, activated the environment source activate geo and then installed conda install -c conda-forge geopandas again to no success.

When creating a new fresh env, it should work (and then it does not matter whether your root environment is based on Anaconda or miniconda). Can you show the full log of what you did (creating new env, installing geopandas, importing it ..) ?

**Comment by ocefpaf** [comment]
Using envs is always recommended regardless of the issue in question.

Also, note that latest shapely added a new logic that tries to find the share library in multiple places. See https://github.com/Toblerity/Shapely/blob/master/shapely/geos.py#L86-L121

It seems that what you are seeing are warnings and not errors when shapely tries to load the shared library from those directories.

> Is there a way we can fix this so that geopandas can successfully install with the default distribution of Anaconda?

Right now? No.

`defaults` and `conda-forge` channels, actually any combinations of channels, have a known and [documented](https://conda-forge.org/docs/conda-forge_gotchas.html) binary incompatibility. The effort to bring those two channels together already started, but it may be months (years?) until we reach 100% compatibility.

Closing this b/c it is unrelated to `geopandas` and not something we can act on.

**Comment by choldgraf** [comment]
If this isn't something `geopandas` can act on, where should people direct their energy when trying to solve this problem? Conda-forge? I ask because I've seen tons of people (e.g. [this one](https://github.com/ResidentMario/geoplot/issues/38#issuecomment-328709417) run into this problem and they're wasting a lot of time trying to solve install conflicts (this is particularly the case on a mac).

I think that if this problem is not solvable, there should be (maybe in docs, in READMEs, etc) instructions on how to deal with this dependency problem. Even if it's just "geopandas can't reliably be installed with the Anaconda distribution on a Mac or Windows, we recommend you use Docker with Ubuntu", or if it linked to these threads, I think it would be helpful for people.

**Comment by ocefpaf** [comment]
> I think that if this problem is not solvable, there should be (maybe in docs, in READMEs, etc) instructions on how to deal with this dependency problem.

There are docs on the matter. Here is a explicit link to them from my comment above: https://conda-forge.org/docs/conda-forge_gotchas.html

**Comment by choldgraf** [comment]
Sure - though for example there's not a link to that page in the docs of geopandas. If you just read the `README`, it'll seem like it's as simple as `pip install geopandas` but we know that's not true for a significant portion of users. (FWIW this is definitely not unique to geopandas, but is pretty much true for every package in the "geo" python ecosystem)

The question isn't whether there are technically docs out there that discuss the problem, it's whether there are docs that frustrated users of our packages will discover on their own. I haven't found anyone using non-linux that has been able to get these packages installed without a fair amount of `conda-forge` hackery...

(that said, this isn't a hill I want to die on...I just think it's a good idea to design documentation that minimizes the time users will waste trying to figure things out)

**Comment by jorisvandenbossche** [comment]
PR to improve the geopandas installation docs is certainly welcome!

**Comment by ocefpaf** [comment]
> Sure - though for example there's not a link to that page in the docs of geopandas. If you just read the `README`, it'll seem like it's as simple as `pip install geopandas` but we know that's not true for a significant portion of users. (FWIW this is definitely not unique to geopandas, but is pretty much true for every package in the "geo" python ecosystem)

OK, but I don't think that is an issue we can address in the `geopandas-feedstock`. This repository is about the `conda` package create by `conda-forge`. As long as people read the `conda-forge` everything should work.

> The question isn't whether there are technically docs out there that discuss the problem, it's whether there are docs that frustrated users of our packages will discover on their own. I haven't found anyone using non-linux that has been able to get these packages installed without a fair amount of `conda-forge` hackery...

Channel preference and the use of envs is not hackery but best practices due to `conda` 's limitation when dealing with third party channels in a more user friendly way, and our (`conda-forge`) limitation to still depend on the default channel for a number of packages.

If you want to raise the issue regarding the Anaconda distribution please do. IMO they should only add a disclaimer to their distribution saying: "Warranty voided if using third party channels."
At the same time they should do an effort to fetch and re-build some key recipes from the community, like this one.

Regarding `conda-forge`, I believe our docs are enough b/c I do see this issue disappearing with time. I maintained the GIS stack from `gdal` to `geopandas` btw.

With that said I can definitely take a PR to our docs to make then less technical as you said.

**Comment by choldgraf** [comment]
oh yeah - when I say documentation, I totally agree that this isn't a conda-forge problem...I'm talking about `geopandas`. I'm happy to open a PR on geopandas as well...something to the effect of:

```
If you are using the `anaconda` distribution, we recommend that you create a new environment
and *only* install packages from the `conda-forge` channel. Installing geospatial packages
(such as `geopandas`) from the default will result in broken links with low-level geospatial libraries.
For example:

    conda create --name myenv
    source activate myenv
    conda install -c conda-forge geopandas
```

**Comment by ocefpaf** [comment]
> oh yeah - when I say documentation, I totally agree that this isn't a conda-forge problem

The fact that you comment here did not make that obvious :wink:

**Comment by choldgraf** [comment]
my apologies - I guess I'm sort of thinking of the conda-forge recipe of geopandas, and geopandas itself, as the same dev team. @jorisvandenbossche can I just append the language above to my PR about sphinx-gallery? Or would you like a new PR?

**Comment by ocefpaf** [comment]
> my apologies

No problem.

> I guess I'm sort of thinking of the conda-forge recipe of geopandas, and geopandas itself, as the same dev team.

That is 2/3 true, but your original message made it sound like something the other 1/3 should fix.

**Comment by choldgraf** [comment]
my bad - it wasn't my intention. I appreciate you being patient with what probably sounded like an unreasonable request!

**Comment by ocefpaf** [comment]
> my bad

Again, no problem :smile: 

> it wasn't my intention. I appreciate you being patient with what probably sounded like an unreasonable request!

Not really. I thought that you your referring to our [docs](https://conda-forge.org/docs/conda-forge_gotchas.html) and I do know that they can be made clearer. But we are biased and cannot easily see the difficult end users have when reading them.

TL;DR if you want to improve https://conda-forge.org/docs/conda-forge_gotchas.html please do!

**Comment by choldgraf** [comment]
TBH I think that the biggest challenge with those docs is one of discoverability rather than content. I don't know of many packages that link to them (which is what I'd recommend for geopandas, e.g.)...that's a tough problem to solve though :-/

**Comment by ocefpaf** [comment]
Actually, we should add a link to them in the README boilerplate in conda-smithy to improve visibility.

**Comment by choldgraf** [comment]
I think that's a good idea!

**Comment by isuruf** [comment]
This, IMHO is a bug in `conda-forge`'s pinning of `libpng`.

**Comment by ocefpaf** [comment]
> This, IMHO is a bug in `conda-forge`'s pinning of `libpng`.

We are doing  `libpng >=1.6.28,<1.7` while defaults seems to be `1.6.27` :unamused: 

Is `1.6.27` forwards compatible? Should we pin to that? I don't recall if we check the ABI laboratory for this.

**Comment by isuruf** [comment]
Can somebody post what `conda list | grep libpng` gives you when you have the error?

**Comment by isuruf** [comment]
And the outputs of `otool -L $CONDA_PREFIX/lib/libpng16.16.dylib` and `otool -L $CONDA_PREFIX/lib/libgdal.dylib`

**Comment by choldgraf** [comment]
FYI I opened a PR to add some language about channels and installing geopandas: https://github.com/geopandas/geopandas/pull/548

**Comment by tbartholomaus** [comment]
@isuruf Below is my conda list output when I get this error:
```ImportError: dlopen(//anaconda/lib/python3.5/site-packages/osgeo/_gdal.cpython-35m-darwin.so, 2): Library not loaded: @rpath/libpng16.16.dylib
  Referenced from: //anaconda/lib/libgdal.20.dylib
  Reason: Incompatible library version: libgdal.20.dylib requires version 45.0.0 or later, but libpng16.16.dylib provides version 39.0.0
```
I had just `conda install gdal`

```[Timothys-MacBook-Pro] ~ $ conda list
# packages in environment at //anaconda:
#
_license                  1.1                      py35_1
_nb_ext_conf              0.3.0                    py35_0
alabaster                 0.7.9                    py35_0
anaconda                  custom           py35hd7b5ba2_0
anaconda-clean            1.0.0                    py35_0
anaconda-client           1.5.1                    py35_0
anaconda-navigator        1.3.1                    py35_0
appnope                   0.1.0                    py35_0
appscript                 1.0.1                    py35_0
argcomplete               1.0.0                    py35_1
asn1crypto                0.22.0                   py35_0
astroid                   1.4.7                    py35_0
astropy                   1.2.1               np111py35_0
babel                     2.3.4                    py35_0
backports                 1.0                      py35_0
backports.functools_lru_cache 1.4                      py35_1    conda-forge
beautifulsoup4            4.5.1                    py35_0
bitarray                  0.8.1                    py35_0
blas                      1.1                    openblas    conda-forge
blaze                     0.10.1                   py35_0
bokeh                     0.12.2                   py35_0
boost                     1.65.1                   py35_0    conda-forge
boost-cpp                 1.65.1                        1    conda-forge
boto                      2.42.0                   py35_0
bottleneck                1.1.0               np111py35_0
bzip2                     1.0.6                         1    conda-forge
ca-certificates           2017.4.17                     0    conda-forge
cairo                     1.14.6                        5    conda-forge
cartopy                   0.15.1                   py35_5    conda-forge
cffi                      1.7.0                    py35_0
chest                     0.2.3                    py35_0
click                     6.6                      py35_0
cloudpickle               0.2.1                    py35_0
clyent                    1.2.2                    py35_0
colorama                  0.3.7                    py35_0
conda                     4.3.29                   py35_0    conda-forge
conda-build               2.0.2                    py35_0
conda-env                 2.6.0                         0    conda-forge
configobj                 5.0.6                    py35_0
contextlib2               0.5.3                    py35_0
cryptography              1.8.1                    py35_0
curl                      7.55.1                        0    conda-forge
cycler                    0.10.0                   py35_0
cython                    0.24.1                   py35_0
cytoolz                   0.8.0                    py35_0
dask                      0.11.0                   py35_0
datashape                 0.5.2                    py35_0
decorator                 4.0.10                   py35_0
dill                      0.2.5                    py35_0
docutils                  0.12                     py35_2
dynd-python               0.7.2                    py35_0
entrypoints               0.2.2                    py35_0
et_xmlfile                1.0.1                    py35_0
expat                     2.1.0                         3    conda-forge
fastcache                 1.0.2                    py35_1
filelock                  2.0.6                    py35_0
flask                     0.11.1                   py35_0
flask-cors                2.1.2                    py35_0
fontconfig                2.12.1                        6    conda-forge
freetype                  2.7                           2    conda-forge
freexl                    1.0.4                         0    conda-forge
future                    0.15.2                   py35_0    conda-forge
gdal                      2.2.3                    py35_0    conda-forge
geos                      3.6.2                         1    conda-forge
geotiff                   1.4.2                         0    conda-forge
get_terminal_size         1.0.0                    py35_0
gettext                   0.19.8.1                      0    conda-forge
gevent                    1.1.2                    py35_0
giflib                    5.1.4                         0    conda-forge
glib                      2.51.4                        0    conda-forge
greenlet                  0.4.10                   py35_0
gsw                       3.0.6                    py35_0    conda-forge
h5py                      2.7.1                    py35_2    conda-forge
hdf4                      4.2.13                        0    conda-forge
hdf5                      1.10.1                        1    conda-forge
heapdict                  1.0.0                    py35_1
icu                       58.2                          0    conda-forge
idna                      2.1                      py35_0
imagesize                 0.7.1                    py35_0
ipykernel                 4.5.0                    py35_0
ipython                   5.1.0                    py35_0
ipython_genutils          0.1.0                    py35_0
ipywidgets                5.2.2                    py35_0
itsdangerous              0.24                     py35_0
jbig                      2.1                           0
jdcal                     1.2                      py35_1
jedi                      0.9.0                    py35_1
jinja2                    2.8                      py35_1
jpeg                      9b                            2    conda-forge
json-c                    0.12.1                        0    conda-forge
jsonschema                2.5.1                    py35_0
jupyter                   1.0.0                    py35_3
jupyter_client            4.4.0                    py35_0
jupyter_console           5.0.0                    py35_0
jupyter_core              4.2.0                    py35_0
kealib                    1.4.7                         4    conda-forge
krb5                      1.14.2                        0    conda-forge
lazy-object-proxy         1.2.1                    py35_0
libdap4                   3.18.3                        2    conda-forge
libdynd                   0.7.2                         0
libffi                    3.2.1                         3    conda-forge
libgdal                   2.2.3                         1    conda-forge
libgfortran               3.0.0                         0    conda-forge
libiconv                  1.15                          0    conda-forge
libkml                    1.3.0                         4    conda-forge
libnetcdf                 4.4.1.1                      10    conda-forge
libpng                    1.6.28                        2    conda-forge
libpq                     9.6.3                         0    conda-forge
libspatialite             4.3.0a                       18    conda-forge
libssh2                   1.8.0                         2    conda-forge
libtiff                   4.0.7                         1    conda-forge
libxml2                   2.9.2                         0
libxslt                   1.1.28                        2
llvmlite                  0.13.0                   py35_0
locket                    0.2.0                    py35_1
lxml                      3.6.4                    py35_0
markupsafe                0.23                     py35_2
matplotlib                2.1.0                    py35_1    conda-forge
mistune                   0.7.3                    py35_1
mkl                       11.3.3                        0
mkl-service               1.1.2                    py35_2
mock                      2.0.0                    py35_0
mpmath                    0.19                     py35_1
multipledispatch          0.4.8                    py35_0
nb_anacondacloud          1.2.0                    py35_0
nb_conda                  2.0.0                    py35_0
nb_conda_kernels          2.0.0                    py35_0
nbconvert                 4.2.0                    py35_0
nbformat                  4.1.0                    py35_0
nbpresent                 3.0.2                    py35_0
networkx                  1.11                     py35_0
nltk                      3.2.1                    py35_0
nose                      1.3.7                    py35_1
notebook                  4.2.3                    py35_0
numba                     0.28.1              np111py35_0
numexpr                   2.6.2                    py35_1    conda-forge
numpy                     1.11.3          py35_blas_openblas_202  [blas_openblas]  conda-forge
obspy                     1.1.0                    py35_1    conda-forge
odo                       0.5.0                    py35_1
olefile                   0.44                     py35_0    conda-forge
openblas                  0.2.19                        2    conda-forge
openjpeg                  2.3.0                         1    conda-forge
openpyxl                  2.3.2                    py35_0
openssl                   1.0.2k                        0    conda-forge
owslib                    0.10.3                   py35_0    scitools
packaging                 16.8                     py35_0    conda-forge
pandas                    0.18.1              np111py35_0
partd                     0.3.6                    py35_0
path.py                   8.2.1                    py35_0
pathlib2                  2.1.0                    py35_0
patsy                     0.4.1                    py35_0
pbr                       1.10.0                   py35_0
pcre                      8.39                          0    conda-forge
pep8                      1.7.0                    py35_0
pexpect                   4.0.1                    py35_0
pickleshare               0.7.4                    py35_0
pillow                    4.3.0                    py35_1    conda-forge
pip                       8.1.2                    py35_0
pixman                    0.34.0                        1    conda-forge
pkginfo                   1.3.2                    py35_0
ply                       3.9                      py35_0
poppler                   0.61.1                        2    conda-forge
poppler-data              0.4.8                         0    conda-forge
proj.4                    4.9.1                         1    scitools
proj4                     4.9.3                         5    conda-forge
prompt_toolkit            1.0.3                    py35_0
psutil                    4.3.1                    py35_0
ptyprocess                0.5.1                    py35_0
py                        1.4.31                   py35_0
pyasn1                    0.1.9                    py35_0
pycosat                   0.6.1                    py35_1
pycparser                 2.14                     py35_1
pycrypto                  2.6.1                    py35_4
pycurl                    7.43.0           py35h69e9701_3
pyepsg                    0.2.0                    py35_0    scitools
pyflakes                  1.3.0                    py35_0
pygments                  2.1.3                    py35_0
pylint                    1.5.4                    py35_1
pyopenssl                 16.2.0                   py35_0    conda-forge
pyparsing                 2.1.4                    py35_0
pyproj                    1.9.5.1                  py35_0
pyqt                      5.6.0                    py35_4    conda-forge
pyshp                     1.2.3                    py35_0    scitools
pytables                  3.4.2                    py35_7    conda-forge
pytest                    2.9.2                    py35_0
python                    3.5.4                         1    conda-forge
python-dateutil           2.5.3                    py35_0
python.app                1.2                      py35_4
pytz                      2016.6.1                 py35_0
pyyaml                    3.12                     py35_0
pyzmq                     15.4.0                   py35_0
qt                        5.6.2                h9e3eb04_4    conda-forge
qtawesome                 0.3.3                    py35_0
qtconsole                 4.2.1                    py35_1
qtpy                      1.1.2                    py35_0
readline                  6.2                           2
redis                     3.2.0                         0
redis-py                  2.10.5                   py35_0
requests                  2.13.0                   py35_0    conda-forge
rope                      0.9.4                    py35_1
ruamel_yaml               0.11.14                  py35_0
scikit-image              0.12.3              np111py35_1
scikit-learn              0.19.0          py35_blas_openblas_201  [blas_openblas]  conda-forge
scipy                     0.19.1          py35_blas_openblas_202  [blas_openblas]  conda-forge
setuptools                27.2.0                   py35_0
shapely                   1.6.2                    py35_1    conda-forge
simplegeneric             0.8.1                    py35_1
singledispatch            3.4.0.3                  py35_0
sip                       4.18                     py35_0
six                       1.10.0                   py35_0
snowballstemmer           1.2.1                    py35_0
sockjs-tornado            1.0.3                    py35_0
spectrum                  0.7.1                    py35_0    conda-forge
sphinx                    1.4.6                    py35_0
spyder                    3.0.0                    py35_0
sqlalchemy                1.0.13                   py35_0
sqlite                    3.13.0                        0
statsmodels               0.6.1               np111py35_1
sympy                     1.0                      py35_0
terminado                 0.6                      py35_0
tk                        8.5.18                        0
toolz                     0.8.0                    py35_0
tornado                   4.4.1                    py35_0
traitlets                 4.3.0                    py35_0
unicodecsv                0.14.1                   py35_0
wcwidth                   0.1.7                    py35_0
werkzeug                  0.11.11                  py35_0
wheel                     0.29.0                   py35_0
widgetsnbextension        1.2.6                    py35_0
wrapt                     1.10.6                   py35_0
xerces-c                  3.2.0                         0    conda-forge
xlrd                      1.0.0                    py35_0
xlsxwriter                0.9.3                    py35_0
xlwings                   0.10.0                   py35_0
xlwt                      1.1.2                    py35_0
xz                        5.2.3                         0    conda-forge
yaml                      0.1.6                         0
zlib                      1.2.11                        0    conda-forge
[Timothys-MacBook-Pro] ~ $`
